ext/standard/basic_functions.stub.php

```diff
@@ -153,11 +153,11 @@ function array_merge(array ...$arrays): array {}
 
 function array_merge_recursive(array ...$arrays): array {}
 
-function array_replace(array $array1, array ...$arrays): array {}
+function array_replace(array $array, array ...$with): array {}
 
-function array_replace_recursive(array $array1, array ...$arrays): array {}
+function array_replace_recursive(array $array, array ...$with): array {}
 
-function array_keys(array $array, mixed $search_value = UNKNOWN, bool $strict = false): array {}
+function array_keys(array $array, mixed $for_value = UNKNOWN, bool $strict = false): array {}
```

Comment:
I liked the original name more.

Reply:
```php
$k = array_keys(array=$arr, search_value='beep');
```
vs
```php
$k = array_keys(array=$arr, for_value='beep');
```

I find the second more descriptive.  "Search" is unclear, since array_keys() doesn't itself imply searching.  "for value" means "keys for this value".

Reply:
I'm with @kocsismate, "for value" means nothing to me.

Reply:
Why was this comment resolved?